apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx

```diff
@@ -0,0 +1,467 @@
+        */
+        useEffect(() => {
+            setCurrentWizardStep(wizStep + 1);
+        }, [ wizStep ]);
+
+        /**
+        * Called when modal close event is triggered.
+        */
+        const handleWizardClose = (): void => {
+
+            // Trigger the close method from props.
+            onWizardClose();
+        };
+        
+        /**
+        * Close the limit reached modal.
+        */
+        const handleLimitReachedModalClose = (): void => {
+
+            setOpenLimitReachedModal(false);
+            handleWizardClose();
+        };
+
+        /**
+        * Callback triggered when the form is submitted.
+        *
+        * @param values - Form values.
+        */
+        const onSubmitWizard = (values: HyprAuthenticationProviderCreateWizardFormValuesInterface): void => {
+
+            const identityProvider: IdentityProviderInterface = { ...template.idp };
+
+            identityProvider.name = values.name.toString();
+            identityProvider.templateId = template.templateId;
+            identityProvider.description = template.description;
+
+            identityProvider.federatedAuthenticators.authenticators[0].properties = [
+                {
+                    "key": "appId",
+                    "value": values.appId.toString()
+                },
+                {
+                    "key": "apiToken",
+                    "value": values.apiToken.toString()
+                },
+                {
+                    "key": "baseUrl",
+                    "value": values.baseUrl.toString()
+                }
+            ];
+
+            if (AppConstants.getClientOrigin()) {
+                if (AppConstants.getAppBasename()) {
+                    identityProvider.image = AppConstants.getClientOrigin() +
+                    "/" + AppConstants.getAppBasename() +
+                    "/libs/themes/default/assets/images/identity-providers/hypr.svg";
+                } else {
+                    identityProvider.image = AppConstants.getClientOrigin() +
+                    "/libs/themes/default/assets/images/identity-providers/hypr.svg";
+                }
+            }
+
+            createNewIdentityProvider(identityProvider);
+        };
+
+        /**
+        * Creates a new identity provider.
+        *
+        * @param identityProvider - Identity provider object.
+        */
+        const createNewIdentityProvider = (identityProvider: IdentityProviderInterface): void => {
+
+            setIsSubmitting(true);
+            createIdentityProvider(identityProvider)
+                .then((response: any) => {
+                    eventPublisher.publish("connections-finish-adding-connection", {
+                        type: componentId
+                    });
+
+                    dispatch(addAlert({
+                        description: t("console:develop.features.authenticationProvider.notifications.addIDP." +
+                        "success.description"),
```

Comment:
Check for indentation. Check other places as well

Reply:
addressed with a new PR